checkstyle-tester/projects-to-test-on-for-github-action.properties

```diff
@@ -32,13 +32,13 @@ apache-struts|git|https://github.com/apache/struts.git|master|
 infinispan|git|https://github.com/infinispan/infinispan|7.2.5.Final||
 protonpack|git|https://github.com/poetix/protonpack|protonpack-1.7||
 jOOL|git|https://github.com/jOOQ/jOOL|version-0.9.7||
-RxJava|git|https://github.com/ReactiveX/RxJava|v1.0.9||
+RxJava|git|https://github.com/ReactiveX/RxJava|3.x|
```

Comment:
@Kevin222004 please point to where we generated report with this project

Reply:
@nrmancuso  https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/c6bab44_2023093734/reports/diff/index.html
this is generated today

Reply:
Good, please always test/show successful execution on projects file (or any file) that we use in CI to make sure that we have not broken anything with update like this.